### ISSUE TITLE
Question on the parameters used in PG4LQR when choosing natural gradient descent

### ISSUE THREAD

**Issue by V-Enzo** [xiangyu-liu/PG4LQR#1]
### Hi xiangyu, thank you for your this project.
I have one question about the parameters you used when you train this model with natural gradient descent.

I have trained it  for around 50000 epochs with state_dim=40, action_dim=10, l=30, m=100, r=0.005 and lr=1e-4 just as the default parameters, but I get this 
![plot result](https://user-images.githubusercontent.com/18146606/78780324-fba0cf80-79d0-11ea-8bb5-780913996509.png)
and we can see the natural gradient mode dosesn't converge,  and I guess it may be the problem of parameter settings. 
Can you tell me what the parameters you used when you train this model under natural gradient descent mode according to the plots you draw in your figures? Thank you!

**Comment by xiangyu-liu** [comment]
I have updated the default setting of my hyper-parameters. I am not sure whether they are the parameters I used to plot my figure. But you can try again on the natural policy gradient version.

**Comment by V-Enzo** [comment]
@xiangyu-liu  Thank you for your update. I have another question about the initial state of the LQR system.  In the algorithm 1 of the _Global convergence of the Gradient Method...._, it says the inital state 
x_0 \in D and  in your implementation, I find that
` self.state = np.random.rand(*[self.state_dim, 1])`, which is used to reset the initial state.
However, since `np.random.seed(0)` only works at the first time when you generate random numbers.
How do we assure that the initial state, the start state of every trajectory, is from the same distribution or can we fix it (give the same value to the initial state of every trajectory) to always roll out from the same initial state?  I am new to control problem and hope to discuss with you. It will be great if you can tell me your e-mail for convenience.

**Comment by xiangyu-liu** [comment]
Our initial state is always sampled from the uniform distribution, which is consistent with the original paper. We set random seeds only to ensure results are always the same when running the same settings.

**Comment by V-Enzo** [comment]
Thank you a lot! I figured it out. @xiangyu-liu
